From 90a51eb7bcb1b61705edb4138a91f7872307c334 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Tue, 28 Aug 2018 17:49:33 +0800
Subject: [PATCH] dt-bindings: thermal: rockchip-thermal: Support the RK1808
 SoCs compatible

This patch set attempts to new compatible for thermal founding
on RK1808 SoCs.

Change-Id: I133218cd958e0aabf711a5d22fe5e5da2fbd59ce
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 Documentation/devicetree/bindings/thermal/rockchip-thermal.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/thermal/rockchip-thermal.txt b/Documentation/devicetree/bindings/thermal/rockchip-thermal.txt
index dfe20de5a117..bee8cf5b4710 100644
--- a/Documentation/devicetree/bindings/thermal/rockchip-thermal.txt
+++ b/Documentation/devicetree/bindings/thermal/rockchip-thermal.txt
@@ -3,6 +3,7 @@
 Required properties:
 - compatible : should be "rockchip,<name>-tsadc"
    "rockchip,px30-tsadc":   found on PX30 SoCs
+   "rockchip,rk1808-tsadc": found on RK1808 SoCs
    "rockchip,rk3228-tsadc": found on RK3228 SoCs
    "rockchip,rk3288-tsadc": found on RK3288 SoCs
    "rockchip,rk3308-tsadc": found on RK3308 SoCs
-- 
2.35.3

